pkg/oci/client/cache/ocicache.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017 Oracle and/or its affiliates. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"syscall"
+
+	baremetal "github.com/oracle/bmcs-go-sdk"
+)
+
+type OCICache struct {
+	vnics    map[string]baremetal.Vnic
+	file     *os.File
+	filename string
+}
+
+// Opne opens the cached and returns the chache handle
```

Comment:
Opne -> Open

Reply:
"Opne opens the cached and returns the chache handle" -> "Open opens the cache and returns the cache handle"